charts/prometheus-to-sd/Chart.yaml

```
@@ -9,4 +9,4 @@ maintainers:
 - name: acondrat
   email: arcadie.condrat@gmail.com
 name: prometheus-to-sd
-version: 0.3.1
+version: 0.3.2
```

I suggest we raise the minor version
```suggestion
version: 0.4.0
```

---

Thanks suggestion!
I made a mistake and committed myself. 205944d